cloud_run_service_connect_cloud_sql/main.tf

```diff
@@ -0,0 +1,114 @@
+# [START cloudrun_service_cloudsql_dbuser_secret]
+resource "google_secret_manager_secret" "dbuser" {
+  project   = "your-project-id"
+  secret_id = "dbusersecret"
+  replication {
+    automatic = true
+  }
+}
+# [END cloudrun_service_cloudsql_dbuser_secret]
+
+# [START cloudrun_service_cloudsql_dbuser_secret_version]
+resource "google_secret_manager_secret_version" "dbuser1" {
```

Comment:
nit: Let's add a comment that this will store secret also as a plain txt in state.
https://registry.terraform.io/providers/hashicorp/google/latest/docs/resources/secret_manager_secret_version